bridges/snowbridge/primitives/router/src/inbound/mod.rs

```diff
@@ -340,17 +346,24 @@ impl<
 		match dest_para_id {
 			Some(dest_para_id) => {
 				let dest_para_fee_asset: Asset = (Location::parent(), dest_para_fee).into();
+				let bridge_location = Location::new(2, GlobalConsensus(network));
 
 				instructions.extend(vec![
+					// After program finishes deposit any leftover assets to the snowbridge
+					// sovereign.
+					SetAppendix(Xcm(vec![DepositAsset {
+						assets: Wild(AllCounted(2)),
```

Comment:
This should really only ever be 1, since we send a different fee to the Parachain but we send the entirety of the asset. However, it doesn't hurt to put a 2 there

Reply:
no, actually it can be two assets:
1. any leftover fee not spent during BuyExecution
2. the transferred asset (which may be different than the fee asset)